### ISSUE TITLE
No decrease parameter at  interface GetOptions (Typescript)

### ISSUE THREAD

**Issue by kornel-kedzierski** [microlinkhq/async-ratelimiter#17]
#### index.d.ts

```
 interface GetOptions {
   id?: string;
   max?: number;
   duration?: number;
   // decrease?: boolean; //// MISSING ////
}
```

#### src/index.js
```
 async get ({
    id = this.id,
    max = this.max,
    duration = this.duration,
    decrease = true
  } = {}) {
```

**Comment by kornel-kedzierski** [comment]
https://github.com/microlinkhq/async-ratelimiter/pull/18

**Comment by kornel-kedzierski** [comment]
fixed in 1.2.2
